plotter/heat.go

```diff
@@ -108,35 +113,27 @@ func (h *HeatMap) Plot(c draw.Canvas, plt *plot.Plot) {
 	var pa vg.Path
 	cols, rows := h.GridXYZ.Dims()
 	for i := 0; i < cols; i++ {
-
 		var right, left float64
 		switch i {
 		case 0:
-			right = (h.GridXYZ.X(i+1) - h.GridXYZ.X(i)) / 2
+			right = (h.GridXYZ.X(1) - h.GridXYZ.X(0)) / 2
```

Comment:
Performance is not an issue here and `/ 2` is easier to read. I'd prefer to leave it if that's OK.